### ISSUE TITLE
Improve "Forgot your password?" screen

### ISSUE THREAD

**Issue by javierarce** [Automattic/wp-calypso#95066]
The current [Forgot your password?](https://wordpress.com/log-in/lostpassword?redirect_to=%2Fsites) page is in a quite messy state.

1. There's a redundant signup link that is also misaligned.
2. The disabled state of the button is not the right one.
3. The description above the form doesn't belong to this screen.
4. The label on top of the input field doesn't follow the capitalization rules of the other login screens

![CleanShot 2024-10-01 at 16 55 29@2x](https://github.com/user-attachments/assets/98bb1a97-8b88-4d80-95a8-eb705eed188b)

#### Steps to reproduce

Visit https://wordpress.com/log-in/lostpassword?redirect_to=%2Fsites or enter your username in the login page and then click the "Forgot password?" link.

#### Proposal

Here's my proposed design to solve the issues I mentioned above:

![image](https://github.com/user-attachments/assets/b985b7a7-ab66-43ff-bc67-2177fbed8a14)

Figma file: wCm5Tzb7JBaG4WNowWbjES-fi-5287_4200

**Comment by Aurorum** [comment]
Out of interest, what's the difference between this page and this one? https://wordpress.com/wp-login.php?action=lostpassword

When I go to https://wordpress.com/log-in/, the "Lost your password?" link leads me there. I'm not sure if it's clear what the difference is - if there is one at all - between a forgotten password and a lost password, especially given the "Lost our password" link on the design above.

**Comment by Copons** [comment]
@escapemanuele this issue is exactly what we observed in our walkthrough! Please use this instead of opening a new issue.

**Comment by escapemanuele** [comment]
Woah that has been bad for who knows how long :D

**Comment by mirka** [comment]
> Out of interest, what's the difference between this page and this one? https://wordpress.com/wp-login.php?action=lostpassword
> 
> When I go to https://wordpress.com/log-in/, the "Lost your password?" link leads me there. I'm not sure if it's clear what the difference is - if there is one at all - between a forgotten password and a lost password, especially given the "Lost our password" link on the design above.

I had the same question, and I also couldn't figure out how you even get to this "Forgot your password" page. Could it be that we should just be linking to the standard "Lost Password" page in every instance, instead of fixing the "Forgot your password" page?

Also linking to a related partial fix that was already made: #96341

**Comment by tellyworth** [comment]
I agree that linking to the [wp-login.php](https://wordpress.com/wp-login.php?action=lostpassword) lost password page instead seems like an easy win for now.

The Calypso version of the login page has a bunch of special branding for various products, but it seems like that branding mostly isn't [carried through to the lost password screen](https://github.com/Automattic/wp-calypso/blob/trunk/client/blocks/login/index.jsx#L406-L432). I'm not sure if that's an oversight or intentional decision.

**Comment by StevenDufresne** [comment]
@chihsuan I see this second lost-password screen was introduced in your PRs: https://github.com/Automattic/wp-calypso/pull/68885, https://github.com/Automattic/wp-calypso/pull/68558. 

The original project thread is: pbIJXs-2mx-p2.

Can you weigh in on this? Do we need to keep both?

**Comment by chihsuan** [comment]
> Can you weigh in on this? Do we need to keep both?

We need to keep the React-based lost password screen for WooCommerce, and also BlazePro flows. When I introduced the screen, it was not intended to be used for WordPress.com. I think you can decide if WordPress.com should use the old or new lost password screen.

This issue seems a duplicate of https://github.com/Automattic/wp-calypso/issues/96291.

**Comment by StevenDufresne** [comment]
I'll close this seeing that #96291 includes more information.
